example/src/simple/component/master.js

```diff
@@ -11,7 +11,7 @@
  *
  */
 
-import utils from './../../../../src/scripts/utils/master';
+import { Component as BasicComponent } from './../../../../src/scripts/component/master';
```

Comment:
I agree, code portability across build engines/platforms should be a primary goal of ours.